mappings/net/minecraft/client/render/GameRenderer.mapping

```diff
@@ -83,5 +145,63 @@ CLASS net/minecraft/class_757 net/minecraft/client/render/GameRenderer
 	METHOD method_3202 shouldRenderBlockOutline ()Z
 	METHOD method_3203 reset ()V
 	METHOD method_3207 disableShader ()V
-	METHOD method_34521 (Lnet/minecraft/class_5912;)V
+	METHOD method_34495 getRenderTypeSolidShader ()Lnet/minecraft/class_5944;
+	METHOD method_34496 getRenderTypeCutoutMippedShader ()Lnet/minecraft/class_5944;
+	METHOD method_34497 getRenderTypeCutoutShader ()Lnet/minecraft/class_5944;
+	METHOD method_34498 getRenderTypeTranslucentShader ()Lnet/minecraft/class_5944;
+	METHOD method_34499 getRenderTypeTranslucentMovingBlockShader ()Lnet/minecraft/class_5944;
+	METHOD method_34500 getRenderTypeTranslucentNoCrumblingShader ()Lnet/minecraft/class_5944;
+	METHOD method_34501 getRenderTypeArmorCutoutNoCullShader ()Lnet/minecraft/class_5944;
+	METHOD method_34502 getRenderTypeEntitySolidShader ()Lnet/minecraft/class_5944;
+	METHOD method_34503 getRenderTypeEntityCutoutShader ()Lnet/minecraft/class_5944;
+	METHOD method_34504 getRenderTypeEntityCutoutNoNullShader ()Lnet/minecraft/class_5944;
+	METHOD method_34505 getRenderTypeEntityCutoutNoNullZOffsetShader ()Lnet/minecraft/class_5944;
+	METHOD method_34506 getRenderTypeItemEntityTranslucentCullShader ()Lnet/minecraft/class_5944;
+	METHOD method_34507 getRenderTypeEntityTranslucentCullShader ()Lnet/minecraft/class_5944;
+	METHOD method_34508 getRenderTypeEntityTranslucentShader ()Lnet/minecraft/class_5944;
+	METHOD method_34509 getRenderTypeEntitySmoothCutoutShader ()Lnet/minecraft/class_5944;
+	METHOD method_34510 getRenderTypeBeacomBeamShader ()Lnet/minecraft/class_5944;
+	METHOD method_34511 getRenderTypeEntityDecalShader ()Lnet/minecraft/class_5944;
+	METHOD method_34512 getRenderTypeEntityNoOutlineShader ()Lnet/minecraft/class_5944;
+	METHOD method_34513 getRenderTypeEntityShadowShader ()Lnet/minecraft/class_5944;
+	METHOD method_34514 getRenderTypeEntityAlphaShader ()Lnet/minecraft/class_5944;
+	METHOD method_34515 getRenderTypeEyesShader ()Lnet/minecraft/class_5944;
+	METHOD method_34516 getRenderTypeEnergySwirlShader ()Lnet/minecraft/class_5944;
+	METHOD method_34517 getRenderTypeLeashShader ()Lnet/minecraft/class_5944;
+	METHOD method_34518 getRenderTypeWaterMaskShader ()Lnet/minecraft/class_5944;
+	METHOD method_34519 getRenderTypeOutlineShader ()Lnet/minecraft/class_5944;
+	METHOD method_34520 getRenderTypeArmorGlintShader ()Lnet/minecraft/class_5944;
```

Comment:
Same questions, `RenderType` should probably be suffix. If mojang named them like `render_type_armor_glint` etc. it's truly cursed and we might need to find better names

Reply:
Mojang named them as such, I simply used the names from the Strings (and thus their file names)